vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -477,6 +483,7 @@ void TextInputShadowNode::updateProperties(winrt::Microsoft::ReactNative::JSValu
   auto hasKeyDownEvents = false;
 
   auto markDirty = false;
+  auto isTextBox = IsTextBox();
   for (auto &pair : props) {
```

Comment:
Should we be making sure that we process `secureTextEntry` first before any other changes?

Reply:
We update the local value here: https://github.com/microsoft/react-native-windows/pull/10356/files#diff-d04c9c359dcc99f741f5d27e172658539f20871b36d8bdae9cae53aa028f0458R537

which is what we were doing before when updating the field on the shadow node, so no net new bugs should be introduced here.